composer.json

```diff
@@ -1,36 +1,44 @@
 {
-	"name": "dnj/laravel-invoice",
-	"license": "MIT",
-	"autoload": {
+    "name": "dnj/laravel-ticketing",
+    "description": "Package used for the ticketing system.",
+    "type": "library",
+    "require": {
+        "php": ">=8.1",
+        "illuminate/support": ">=5.4"
```

Comment:
I see two scenario:
1- if you are in production environment (that means require-dev packages are not installed) then you sure than this package is running in a laravel project and `laravel/laravel` package already installed.
2- if you are in development environment (this means `orchestra/testbench` installed) then sou sure laravel/laravel is installed too.
So there is no point to bother ourselves with laravel/* and illuminate/* dependencies.